### ISSUE TITLE
Cannot get WoW realm status

### ISSUE THREAD

**Issue by olorton** [LogansUA/blizzard-api-php-client#26]
The url for this seems to be wrong:

Currently:
```php
    public function getRealmStatus(array $options = [])
    {
        return $this->request('/realm/', $options);
    }
```
I believe this should be:
```php
    public function getRealmStatus(array $options = [])
    {
        return $this->request('/realm/status', $options);
    }
```

I'd be happy to submit a PR once this is verified as an issue

**Comment by LogansUA** [comment]
@olorton Hello. Thank you, waiting for PR.

**Comment by LogansUA** [comment]
Fixed by #27

**Comment by olorton** [comment]
@LogansUA Thanks for fixing this. I've not had a chance to create the PR.
